src/outofprocerrorwrapper.ts

```diff
@@ -0,0 +1,11 @@
+import { OrchestratorState } from "./classes";
+
+const outOfProcDataLabel = "\n\n$OutOfProcData$:";
+
+export class OutOfProcErrorWrapper extends Error {
```

Comment:
More naming nits: 
* I suggest the suffix `Result` rather than `Wrapper` to make it clear that this data is used as the result of a function execution.
* I suggest we avoid the term "out-of-proc" since it has no meaning from the point of view of a JavaScript app (which does not know or care that it is running "out-of-proc" - only the extension cares about this).

Perhaps a name like `DurableErrorResult` would be more appropriate?

Reply:
I am hesitant to call it `DurableErrorResult`, as that implies this only applies to `DurableError`s, when it applies to all errors, including those thrown by the customer.

The name I am leaning towards is `OrchestrationFailureError`. The only error that should be thrown by `orchestrator.handle()` should be this type, and I feel this name captures that.